src/api/v2/account_activities.rs

```diff
@@ -210,8 +206,8 @@ pub struct NonTradeActivityImpl<T> {
   pub type_: T,
   /// The date on which the activity occurred or on which the
   /// transaction associated with the activity settled.
-  #[serde(rename = "date", deserialize_with = "system_time_from_date_str")]
-  pub date: SystemTime,
+  #[serde(rename = "date")]
+  pub date: NaiveDate,
```

Comment:
I'm not sure if this is the right solution. Alpaca seems to only send `YYY-MM-DD` here, not a full timestamp. I think we could do `DateTime<Utc>` here and do a custom parser for Serde.

Reply:
Yes, agreed. I don't believe that `NonTradeActivityImpl` is serializable, so the last point should not be necessary.